### ISSUE TITLE
Backdrop filter doesn't work on child when also applied to parent

### ISSUE THREAD

**Issue by vedmant** [tailwindlabs/tailwindcss#5825]
**What version of Tailwind CSS are you using?**

v2.2.1

**What build tool (or framework if it abstracts the build tool) are you using?**

NA

**What version of Node.js are you using?**

NA

**What browser are you using?**

Chrome

**What operating system are you using?**

NA

**Reproduction URL**

https://jsfiddle.net/tzy8ne7v/20/

**Describe your issue**

When "backdrop-filter backdrop-blur-sm" applied to parent  and child block it works only on parent one.

**Comment by adamwathan** [comment]
Hey! This is just a limitation of how Chromium browsers handle backdrop-filters. You can see your demo _does_ work in Safari:

![image](https://user-images.githubusercontent.com/4323180/137893268-3a58e6cd-90ce-444c-86ad-448965b34e45.png)

And even using vanilla CSS for the blur doesn't work in Chrome:

https://jsfiddle.net/pjzb27h8/

So this is just a CSS/browser issue I'm afraid, impossible for us to do anything about it. We give you the classes to set the CSS property, what the browser does when given that property is unfortunately out of our control.

Hope that helps!

**Comment by vedmant** [comment]
@adamwathan Thanks for the response! I just also figured this out, this ticket may help someone else if they encounter the same problem.

**Comment by narr07** [comment]
Do you have solution for this? @vedmant

**Comment by vedmant** [comment]
@narr07 

> This is just a limitation of how Chromium browsers handle backdrop-filters.

**Comment by joaopaulomoraes** [comment]
`F` for Chrome!

**Comment by acalvino4** [comment]
Just ran into this issue. I see there's no solution or workaround posted, but curious if anyone has a link to a chromium bug report on it.

**Comment by acalvino4** [comment]
Found the "bug" report: https://bugs.chromium.org/p/chromium/issues/detail?id=993644

TLDR: Interestingly, chromium's behavior is actually according to spec, and it's safari that deviates.

For a workaround see [this stackoverflow](https://stackoverflow.com/a/76207141).

**Comment by mohamedtsx** [comment]
For a workaround solution apply the filter to a pseudo element.

**Comment by aervxa** [comment]
> TLDR: Interestingly, chromium's behavior is actually according to spec, and it's safari that deviates.

@acalvino4 Interestingly enough, it works on Firefox, and the `:before` pseudo elm is a workaround to support chrome, since it's, yk, *chrome*.
